pom.xml

```diff
@@ -156,7 +156,7 @@
     <flink.clients.artifactId>flink-clients</flink.clients.artifactId>
     <flink.connector.kafka.artifactId>flink-connector-kafka</flink.connector.kafka.artifactId>
     <flink.hadoop.compatibility.artifactId>flink-hadoop-compatibility_2.12</flink.hadoop.compatibility.artifactId>
-    <rocksdbjni.version>5.17.2</rocksdbjni.version>
+    <rocksdbjni.version>7.5.3</rocksdbjni.version>
```

Comment:
does RocksDB `5.17.2` not work?  Dependency version upgrade has larger impact.

Reply:
Right, RocksDB `5.17.2` would throw `NoClassDefException` when running `TestHoodieLogFormat`

Reply:
This seems fishy.  Does this only happen for Java 17?

Reply:
Yes, for Java 8 it works fine. This issue only pops up for Java 17

Reply:
After reading the code, `RocksDbBasedFileSystemView` is the only place using RocksDb.  So this should be ok.